README.md

```diff
@@ -115,9 +148,9 @@ val results = for (_ <- Range(0, 10).par)
     yield submitter.runOnCluster(read, readOnlyEnv)
```

Comment:
its a bit unclear where the submitter is coming from in this example. I would still include the `withSubmitter` function since this is an independent example from the previous one.